### PR TITLE
ADD FAQ (1 - 5 )

### PR DESCRIPTION
**Pull request by s950449**
Add faq

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -227,8 +227,26 @@ Some examples:
...
 
 * Send a PR and solve a bug in this repository (30%)
-
-
+##FAQ
+1.Python Environment
+    1a.(Recommended) If you are using miniconda, please make sure that you have create the environment 
+    By using `conda env create -f environment.yaml`, you can create an environment to meet the need for this homework.
+    To activate your environment:`source activate your_environment_name` 
+    1b.If you have problem or can't install miniconda (e.g. Arm-based linux such as Raspbian) ,please make sure that Python3 version is 3.5 or later.
```
[the remark this answers is not in the document]
Comment (amjltc295) [reply README.md]: Also a typo `) ,` -> `), `

File: README.md [hunk 2]
```diff
@@ -227,8 +227,26 @@ Some examples:
...
+    You could use `python3 -V` to check your Python version. And if you want to install python package, you could use `pip3 install <package_name>`
```
Comment (amjltc295) [inline README.md]: Install packages globally is not a good habit. If you could not install Conda, please take a look at virtual environment https://docs.python.org/3/tutorial/venv.html

Comment (amjltc295) [inline README.md]: Installing packages globally is not a good habit. If you could not install Conda, please take a look at virtual environment https://docs.python.org/3/tutorial/venv.html

File: README.md [hunk 3]
```diff
@@ -227,8 +227,26 @@ Some examples:
...
+
+2.In order to pass Task 8, please make sure that you have already install the font 'arial' or arial.ttf has already in your system's font folder.
+If you are using Debian/Ubuntu, you can install the font 'arial' by using apt:
+`sudo apt install ttf-mscorefonts-installer`
+`sudo fc-cache`
+To check whether the font has successfully installed, you can use `fc-match Arial`
+
+3.In task 8, due to utils.py and your_student_id.py in different directory, you should add `sys.path.append('../')` in your file.
```
[the remark this answers is not in the document]
Comment (s950449) [reply README.md]: I rum autograder.py on `PythonHomework/src`

File: README.md (hunk 2 above)
Comment (s950449) [reply README.md]: I got it.
Useful tips

**Pull request closed without merging**